LegacyComponents/LegacyComponentsInternal.m

```diff
@@ -75,6 +75,9 @@ int iosMajorVersion()
             case 12:
                 version = 12;
                 break;
+            case 13:
```

Comment:
BTW this one is "irrelevant", came from another commit. So feel free to discard this lines :)